x-pack/plugins/rollup/public/crud_app/sections/components/field_list/field_list.js

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+
+import {
+  EuiInMemoryTable,
+  EuiEmptyPrompt,
+} from '@elastic/eui';
+
+export const FieldList = ({
+  columns,
+  fields,
+  onRemoveField,
+  addButton,
+  emptyMessage,
+}) => {
+  let message;
+
+  if (!fields.length) {
+    message = (
+      <EuiEmptyPrompt
+        title={emptyMessage}
+        titleSize="xs"
+      />
+    );
+  }
+
+  let extendedColumns;
+
+  if (onRemoveField) {
+    extendedColumns = columns.concat({
+      name: 'Remove',
+      width: '80px',
+      actions: [{
+        name: 'Remove',
```

Comment:
If this field is displayed on UI, please translate it. The same for `description`.